src/autora/state/wrapper.py

```diff
@@ -85,3 +88,166 @@ def experimentalist(params):
         return Delta(conditions=conditions_)
 
     return experimentalist
+
+
+class WrapperInputError(Exception):
+    pass
+
+
+def wrap_to_use_state_general(
```

Comment:
Could the name be shorted, e.g., ``wrap_to_state``? I imagine people will be using this a lot, so we might want to give it a short name.